chisel/src/bin/chisel.rs

```diff
@@ -132,6 +135,11 @@ async fn main() -> eyre::Result<()> {
     let mut rl = Editor::<SolidityHelper, _>::new()?;
     rl.set_helper(Some(SolidityHelper::default()));
 
+    // load history
+    if let Some(chisel_history) = chisel_history_file() {
+        let _ = rl.load_history(&chisel_history);
+    }
+
```

Comment:
This probably needs [set_auto_add_history](https://docs.rs/rustyline/latest/rustyline/config/trait.Configurer.html#method.set_auto_add_history)?

Reply:
hmm we already add it manually

https://github.com/foundry-rs/foundry/blob/7ef5d11750a6157ef46041bec264b9e5760ee4bf/chisel/src/bin/chisel.rs#L163

I geuss we can remove `add_entry` and enable auto add instead?

Reply:
made that change real quick